src/renderer/families/tron/Votes/index.js

```diff
@@ -85,7 +86,11 @@ const Delegation = ({ account, parentAccount }: Props) => {
   const superRepresentatives = useTronSuperRepresentatives();
   const nextVotingDate = useNextVotingDate();
 
-  const { tronResources: { votes, tronPower, unwithdrawnReward } = {} } = account;
+  const unit = getAccountUnit(account);
+  /** min 1TRX transactions */
+  const minAmount = 10 ** unit.magnitude;
+
+  const { tronResources: { votes, tronPower, unwithdrawnReward } = {}, spendableBalance } = account;
```

Comment:
i'm not fan of this magic code, I think it would be way more readable to do

```js
  const { tronResources, spendableBalance } = account;
  const { votes, tronPower, unwithdrawnReward } = tronResources || {};
```

is there a reason why `tronResources` would be falsy? i assume it would always be here, and if so I recommend to do `invariant(tronResources, "")` because i don't understand how flow can be happy here.

Reply:
You can get no tron resources if you're in a token account i would not use invariant for this reason this piece of UI would trigger an error and stop the app from showing the token account page :+1: